### DISCUSSION TITLE
[Status Bars] Option to always show in Wintertodt when outside of combat

### DISCUSSION THREAD

**Discussion by darkscion0** [runelite/runelite#18278] [Ideas]
I love the option to hide the bars after x ticks of leaving combat as it saves me from having to turn off the plugin when hiding them during non-combat skilling like Tithe Farm.

That being said, it would be nice to have the option to always show them when entering Wintertodt as they allow the tracking of your health much easier.

**Comment by YvesW** [comment]
Wt warmth has been added in https://github.com/runelite/runelite/commit/95d4208ade15c9df622ab8c71da2897e10abbd3a
However, I'm not fully sure if this does solve your issue as it will likely not count warmth draining as taking damage with the current implementation.
